src/pkg/term/colorizer_test.go

```
@@ -162,6 +162,35 @@ func TestIsTerminal(t *testing.T) {
 	}
 }
+
+// TestPrintRoutingInJSONMode is a regression test: Print/Println/Printf/Printc
+// used to always write to stdout, so any of them reachable from a command
+// that also emits --json output (e.g. via a debug dump) would corrupt that
+// JSON payload. They must move to stderr in JSON mode, like Info/Warn do.
+func TestPrintRoutingInJSONMode(t *testing.T) {
+	var stdout, stderr bytes.Buffer
+	defaultTerm := NewTerm(os.Stdin, &stdout, &stderr)
+
+	defaultTerm.Print("a")
```

golangci-lint (2.12.2)

[error] Error return value of `defaultTerm.Print` is not checked

(errcheck)